webpack.config.dev.babel.js

```diff
@@ -0,0 +1,62 @@
+import path from 'path';
+import merge from 'webpack-merge';
+import webpack from 'webpack';
+import commonConfig from './webpack.config.common';
+import { version } from './package.json';
+
+const config = merge(commonConfig, {
+  mode: 'development',
+  entry: [
+    'react-hot-loader/patch',
+    'webpack-dev-server/client?http://localhost:8090',
+    'babel-polyfill',
+    './index.js'
```

Comment:
For the case-sensitive machines (like mine) you made a mistake, cause your file is basically `Index.js`. So that initially I wasn't able to run you app :) Be careful :+1: